backend/src/main/java/org/launchcode/fancyrats/models/Job.java

```diff
@@ -0,0 +1,108 @@
+package org.launchcode.fancyrats.models;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+
+import java.time.LocalDate;
+
+@Entity
+public class Job {
+
+    @Id
+    @GeneratedValue
+    private int id;
+
+    private int zipCode;
+
+    private LocalDate startDate;
+
+    private LocalDate endDate;
+
+    //final?
+    private LocalDate createdDate;
+
+    private long payRate;
+
+    private String description;
+
+    @ManyToOne
+    private Client client;
```

Comment:
Hey @Bryanary-code, let me know what you're thinking about for the user model. I'm adding a comment here so I remember where to swap it out of the Job model :)

Reply:
Hey, I'm thinking:  id
                              username
                              password
                              email
                              name 
                              address
                              phone number
                              role (pet sitter, pet owner, both)
                              pet type(s)
I think these should suffice for a basic user profile. Then allow for more customization after account creation ( i.e. photos, bio, reviews, social links, etc.) What do you think?